src/lib/rest/StringFilter.cpp

```diff
@@ -1922,12 +1922,38 @@ bool StringFilter::mqMatch(ContextElementResponse* cerP)
     ContextAttribute*  caP   = cerP->contextElement.getAttribute(itemP->attributeName);
     Metadata*          mdP   = (caP == NULL)? NULL : caP->metadataVector.lookupByName(itemP->metadataName);
 
-    if ((itemP->op == SfopExists) && (mdP == NULL))
+    if ((itemP->op == SfopExists) || (itemP->op == SfopNotExists))
```

Comment:
Now, the logic is the same as the one usef for qMatch()

NTC (just informative)